css/_workinghourspicker.scss

```diff
@@ -72,13 +72,20 @@
 	}
 }
 
-//old work hour picker
-.working-hours-radio{
-  margin-top: $full-padding;
-  display: flex;
-  display: -ms-flexbox;
-}
-
-.working-hours-row{
+//Old working-hours-picker
+.working-hours-row {
 	border-top: 1px solid $gray012;
-}
+
+	& > .con-flex {
+		align-self: center;
```

Comment:
do we need to make a change to the con-flex here? would it be nicer to have a new class for this?

Reply:
Ehm... it will only change `.con-flex` classes whose parent is `.working-hours-row`. But I can add a new `.working-hours-day` class and move the declaration.

Reply:
Thanks, will merge it when the checks are done.
its more about what if we want to add something else in a con-flex  :)